src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -37,26 +37,31 @@
         data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
         {% block bodyattrs %}{% endblock %}>
 
-    <nav class="DevHub-Navigation">
+    <nav class="DevHub-Navigation {% if request.user.is_authenticated() %}DevHub-Navigation--signedin{% endif %}">
       <ul>
         <li><a href="#"><div class="Logo" alt="{{ dev_page_title() }}" /></a></li>
         <li><a href="#">Docs</a></li>
         <li><a href="#">Support</a></li>
         <li><a href="#">Blog</a></li>
+        {% if request.user.is_authenticated() %}
+          <li><a href="#"><img class="avatar" src="{{ request.user.picture_url }}" /></a></li>
```

Comment:
One thing to bear in mind here is there's no concept of a mobile user's settings page (as far as the mobile pages v1 is concerned). So this link should probably log people out. That can be a separate issue of course.